src/ol/interaction/drawinteraction.js

```diff
@@ -624,5 +643,7 @@ ol.interaction.Draw.getMode_ = function(type) {
 ol.interaction.DrawMode = {
   POINT: 'Point',
   LINE_STRING: 'LineString',
-  POLYGON: 'Polygon'
+  POLYGON: 'Polygon',
+  BOX: 'Box',
```

Comment:
Ah, that shouldn't be there. It was left over from my first attempt to add both box and circle. I'll rebase to remove that.